### ISSUE TITLE
Time for a new header image?

### ISSUE THREAD

**Issue by sn3p** [DefactoSoftware/website#437]
I think we could freshen up the website a little, e.g. by using a new header image. Ideas are welcome!

Here's a quick experiment using a gradient (love the orange/purpelish complementary colors):

<img width="861" alt="screen shot 2016-12-22 at 12 29 59" src="https://cloud.githubusercontent.com/assets/935529/21425154/bd07ca6e-c847-11e6-9535-cfad0e73e7cf.png">

**Comment by RonaldDijkstra** [comment]
I like this, could be on more pages than just home.

**Comment by sn3p** [comment]
Yes sounds good. Can tweak the gradient a bit, and maybe different colors for different pages.
